### PR TITLE
feat: unreal engine 4 support

### PR DESCRIPTION
**Pull request by bruno-garcia**
* New endpoint: `https://sentry.io/api/4/unreal/pub_key/`
* Access `symbolic` to retrieve unreal files from within uploaded dump.
* Processes minidump file from crash.
* Attaches all files if attachments feature flag is on.

### CODE REVIEW

File: requirements-base.txt [hunk 1]
```diff
@@ -62,7 +62,7 @@ sqlparse>=0.1.16,<0.2.0
 statsd>=3.1.0,<3.2.0
 strict-rfc3339>=0.7
 structlog==16.1.0
-symbolic>=5.5.6,<6.0.0
+symbolic>=5.7.0,<6.0.0
```
Comment (bruno-garcia) [inline requirements-base.txt]: @jan-auer build's broken still looking for 5.5.6 for some reason

Comment (bruno-garcia) [reply requirements-base.txt]: Travis cache!

**Review by jan-auer — APPROVED** [review]
Looks good. getsentry has been updated to `symbolic==5.7.0`, so this is good to go.

**Comment by mitsuhiko** [comment]
Marge it

**Review by mitsuhiko — APPROVED** [review]

File: tests/sentry/lang/native/test_unreal.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    file = MockFile("minidump")
+    assert unreal_attachment_type(file) == MINIDUMP_ATTACHMENT_TYPE
+
+
+def test_unreal_attachment_type_unknown():
+    file = MockFile("something unknown")
+    assert unreal_attachment_type(file) is None
+
+
+class MockFile():
```
Comment (mitsuhiko) [inline tests/sentry/lang/native/test_unreal.py]: This should subclass `object` but it’s no blocker.

**Pull request merged**